elements/lisk-cryptography/test/bls_lib/lib.spec.ts

```diff
@@ -58,6 +60,29 @@ interface EthFastAggrVerifySpec {
 }
 
 describe('bls_lib', () => {
+	const curveOrder = '0x73eda753299d7d483339d80809a1d80553bda402fffe5bfeffffffff00000001';
+
+	describe('isSecretKeyNonZeroModEC', () => {
+		it('should return true when given a valid secret key', () => {
+			const secretKey = SecretKey.fromBytes(Buffer.alloc(32, 1));
+			expect(isSecretKeyNonZeroModEC(secretKey)).toBe(true);
+		});
+
+		it('should return true when given a non-zero modulo secret key', () => {
```

Comment:
I think we should add test case for zero secret key?

Reply:
That would be a nice check but I think the functionality encapsulated in this function is only to check that a secret key being used for signing and other operations is non-zero modulo the order of the group. Checks for zero secret key are carried out in conjunction with `isSecretKeyNonZeroModEC` function. Thoughts?

Reply:
can be resolved from my side